spark_pipeline_framework/transformers/fhir_receiver/v1/fhir_receiver.py

```diff
@@ -472,6 +484,17 @@ def _transform(self, df: DataFrame) -> DataFrame:
             self.refresh_token_function
         )
 
+        if parameters and parameters.get("flow_name"):
+            user_agent_value = (
```

Comment:
we could have write this in short way
user_agent_value = parameters["flow_name"]
    if parameters.get("team_name"):
        user_agent_value += f"/{parameters['team_name']}"
    additional_request_headers["User-Agent"] = user_agent_value
    
please change everywhere